### ISSUE TITLE
TrabalhoRoboutler_X,Y, coordinate goal

### ISSUE THREAD

**Issue by Bereke23** [miguelriemoliveira/psr_22-23#13]
We have a question regarding the autonomous navigation with specific (X,Y) coordinates. Does using the RVIZ "2D nav goal" feature qualify as an approved method for setting coordinate target?
Thanks in advance

**Comment by miguelriemoliveira** [comment]
Hi @Bereke23 ,

> Does using the RVIZ "2D nav goal" feature qualify as an approved method for setting coordinate target

I am not sure I understand your question. An approved method by the evaluation rules?
Yes, its ok that you use the 2D nav goal.

**Comment by Tisila** [comment]
Exactly, the question was regarding if the existing solution is approved by the evalution rules.
Thanks

**Comment by miguelriemoliveira** [comment]
You can use it of course. But notice that is not all that is asked. I also ask for some sort of semantic specification of the destination in a mission such as "robutler, go to the kitchen".

**Comment by Tisila** [comment]
Indeed, we will develop a semantic target solution.
